json-unit-core/src/main/java/net/javacrumbs/jsonunit/core/internal/Diff.java

```diff
@@ -62,12 +63,12 @@
  */
 public class Diff {
 
-    private static final Pattern ANY_NUMBER_PLACEHOLDER = Pattern.compile("[$#]\\{json-unit.any-number\\}");
-    private static final Pattern ANY_BOOLEAN_PLACEHOLDER = Pattern.compile("[$#]\\{json-unit.any-boolean\\}");
-    private static final Pattern ANY_STRING_PLACEHOLDER = Pattern.compile("[$#]\\{json-unit.any-string\\}");
+    private static final Pattern ANY_NUMBER_PLACEHOLDER = Pattern.compile("[$#]\\{json-unit.any-number}");
```

Comment:
Please do not do this, it breaks Android tests https://github.com/lukas-krecan/JsonUnit/pull/227